src/App.svelte

```diff
@@ -20,10 +47,11 @@
   }
 
   page("/", setComponent(AppList));
-  page("/apps/:app/tables/:ping", setComponent(TableDetail));
+  page("/apps/:app/tables/:ping/:bigquery", setComponent(TableDetail));
   page("/apps/:app/pings/:ping", setComponent(PingDetail));
   page("/apps/:app/metrics/:metric", setComponent(MetricDetail));
   page("/apps/:app", setComponent(AppDetail));
+  page("*", setComponent(Breadcrumb));
```

Comment:
Did you mean to add this? It seems like an error and might conflict with some other work that's happening.

Reply:
Oh, my bad. I wanted it to be removed.